src/func/Microsoft.Health.Fhir.Ingest.Host/IomtConnectorFunctions.cs

```diff
@@ -25,10 +25,17 @@ namespace Microsoft.Health.Fhir.Ingest.Service
     public class IomtConnectorFunctions
     {
         private readonly ITelemetryLogger _logger;
+        private readonly CollectionContentTemplateFactory _collectionContentTemplateFactory;

```

Comment:
>CollectionContentTemplateFactory [](http://example.com/codeflow?start=25&length=32)

Have we done an e2e test to verify the data flow for the old Azure functions?

Reply:
No I have not, but I will do that

Reply:
Deployed Azure Functions using `Create-IomtFhirSandboxEnvironment` script. Submitted events to event hub and noted that NormalizationProcessor successfully processed the events. I didn't notice the events flowing into the FhirConverter Function, or into the NormalizedData event hub. I don't see any errors, but will continue to explore this.


![image](https://user-images.githubusercontent.com/71304426/127581475-00f43d06-06cd-4e29-a789-643142a90985.png)

Reply:
Turned out to be a misconfigured device content template. Now data flows from normalization -> fhirconversion -> fhir

![image](https://user-images.githubusercontent.com/71304426/127681778-aba19198-ef74-4633-b058-78b34f867d20.png)

![image](https://user-images.githubusercontent.com/71304426/127681963-6b920d8b-4dde-4c08-b236-699eda223da7.png)